draft-pda-protocol.md

```diff
@@ -450,29 +450,16 @@ element.)]
 
 ### Upload Start
 
-The client sends a POST request to `[leader]/upload_start` with the following
-message:
-
-~~~
-struct {
-  PATask task;
-} PAUploadStartReq;
-~~~
-
-The `task` field corresponds to the PA task for which a report will be
-generated.
-
-The leader responds to well-formed requests to `[leader]/upload_start` with
-status 200 and the following message:
+The client sends a GET request to `[leader]/[version]/[task_id]/upload_start`,
+where `[version] == PATask.version` and `[task_id] == PATask.id`. [TODO: Decide
+if adding the protocol version and task id to the URL is the right way to go.
+Instead, we may want to make this a POST with a body that specifies the PATask.
```

Comment:
Our experience with caching OCSP for Let's Encrypt has been that caching POST responses is tricky because CDNs don't always support this as well as caching GETs. However we can resolve that question in #61.